app/controllers/transactor/TransactorCaptureEmailPasscodeController.scala

```diff
@@ -104,10 +105,11 @@ class TransactorCaptureEmailPasscodeController @Inject()(view: CaptureEmailPassc
   }
 
   private def getEmailAddress(implicit hc: HeaderCarrier, profile: CurrentProfile): Future[String] = {
+    val missingAnswerSection = "tasklist.aboutBusinessContact.contactDetails"
```

Comment:
should be tasklist.aboutYou.contactDetails as the transactor is "You" in transactor flows